cookbooks/aws-parallelcluster-scheduler-plugin/files/default/sudo_set_home/1_parallelcluster

```diff
@@ -0,0 +1 @@
+Defaults always_set_home
```

Comment:
is this setting applied for all the users or just root?

Reply:
it is for all the user who run sudo command, it set HOME to /root